.github/workflows/test.yml

```diff
@@ -150,6 +171,13 @@ jobs:
           name: Slow unit test results
           path: "**/target/surefire-reports/"
           retention-days: 7
+      - name: Archive Test Errors
+        uses: actions/upload-artifact@v3
+        if: failure()
+        with:
+          name: Slow unit test errors for ${{ matrix.project }}
```

Comment:
❌ This job does not contain a matrix so we shouldn't reference it in the name of the archive.